README.rst

```diff
@@ -1,6 +1,191 @@
-===================
-services-marionette
-===================
-
+==================
+marionette-wrapper
+==================
+.. image:: https://img.shields.io/pypi/l/fxapom.svg
+   :target: https://github.com/mozilla-services/services-marionette/blob/master/LICENSE
+   :alt: License
 .. image:: https://travis-ci.org/mozilla-services/services-marionette.svg?branch=master
     :target: https://travis-ci.org/mozilla-services/services-marionette
+.. image:: https://img.shields.io/github/issues-raw/mozilla-services/services-marionette.svg
+   :target: https://github.com/mozilla-services/service-marionette/issues
+   :alt: Issues
+.. image:: https://img.shields.io/requires/github/mozilla-services/marionette-wrapper.svg
+   :target: https://requires.io/github/mozilla-services/marionette-wrapper/requirements/?branch=master
+   :alt: Requirements
+
+Overview
+--------
+This package contains a class that serves as a base class for page object classes. The base class wraps commonly used
+`Selenium WebDriver <http://docs.seleniumhq.org/docs/03_webdriver.jsp>`_ functionality into readable functions and gives a clean API for page objects to consume.
+
+Installation
+------------
+
+:code:`python setup.py develop`
+
+Usage
+-----
+
+To use the base class, you would create a page object class that inherits from the base class found in base.py. View the API Reference
+for the exact calls that can be made in the API.
+
+.. code-block:: python
+
+    from services-marionette.base import Base
+
+    class PageObject(Base):
+        #code
+
+Run Tests
+~~~~~~~~~
+The sample test included with this package uses pytest to run the test. To invoke the test run the following commmand:
+
+:code:`py.test --capture=sys tests/test_marionette_wrapper_commands.py`
+
+API Examples For Common Functionality
+-------------------------------------
+
+This reference shows the marionette-base API call and it's equivalent Selenium counterpart.
```

Comment:
Probably worth a bit of manual search+replace action, as I think we call it "marionette-base" in about 12 places currently.